src/main/java/hu/wurfel/refference/school/diary/DiaryController.java

```diff
@@ -24,23 +24,23 @@ public ResponseEntity<List<Diary>> search(@PathVariable EntityNames searchWith,
 	@PostMapping("/api/v1/Diary/adding")
 	public ResponseEntity<List<Diary>> adding(@RequestBody DiaryRequestForSearch diaryRequestForSearch) {
 		List<Diary> answer = new ArrayList<>();
-		answer.add(diaryService.saveDiary(Integer.parseInt(diaryRequestForSearch.getId()), Long.parseLong(diaryRequestForSearch.getStudentId()), Integer.parseInt(diaryRequestForSearch.getClassId())));
+		answer.add(diaryService.save(Integer.parseInt(diaryRequestForSearch.getId()), Long.parseLong(diaryRequestForSearch.getStudentId()), Integer.parseInt(diaryRequestForSearch.getClassId())));
 		return ResponseEntity.ok(answer);
 	}
 
 	@DeleteMapping("/api/v1/Diary/delete/{id}")
 	public void deleteById(@PathVariable String id) {
-        diaryService.deleteDiary(diaryService.getDiaryByDiaryid(Integer.parseInt(id)));
+		diaryService.delete(diaryService.getByDiaryid(Integer.parseInt(id)));
```

Comment:
Valamelyik commentben említettem, hogy a xServiceben simán lehet a hozzá tartozó repository field neve repo. Ez igaz a controllerre is. A DiaryControllerben a DiaryService mezőneve lehet simán service. Szintén nem közelező, de szerintem célszerűbb.

Reply:
az InteliJ ajánlotta ezt a nevet auto mindenhol, azért használom ezt

Reply:
Ez egyáltalán nem hiba, csak javaslat. 😺